app/models/album.rb

```diff
@@ -0,0 +1,6 @@
+class Album < ActiveRecord::Base
+  validates :title, :genre, :artist, :length, presence: true
+  validates :length, numericality: true
```

Comment:
If I get specific with a validation like this, I'll take the presence validation off of the chained together line and add it here. Also, what happens if length is negative?
```ruby
validates :length, numericality: true, presence: true
```